<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="login_tab">
    <attribute name="title">
     <string>Login</string>
    </attribute>
    <widget class="QLineEdit" name="nickname_field">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>80</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string> Nickname</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_field">
     <property name="geometry">
      <rect>
       <x>54</x>
       <y>140</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string> Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>85</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_noti">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>57</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check your password or nickname&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Unable To Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="chat_tab">
    <attribute name="title">
     <string>Chat</string>
    </attribute>
    <widget class="QPlainTextEdit" name="msgs_box">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>20</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="new_msg_field">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>320</y>
       <width>501</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="send_btn">
     <property name="geometry">
      <rect>
       <x>528</x>
       <y>319</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settings_tab">
    <attribute name="title">
     <string>Settings</string>
    </attribute>
    <widget class="QPushButton" name="logout_btn">
     <property name="geometry">
      <rect>
       <x>258</x>
       <y>310</y>
       <width>85</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
